src/main/java/bookstore/model/ShoppingCart.java

```diff
@@ -0,0 +1,35 @@
+package bookstore.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "shopping_carts")
+@Getter
+@Setter
+public class ShoppingCart {
```

Comment:
Should it be a soft-deleted entity?

Reply:
I think not because I don`t need saved shopping carts after deleting them. Instead, I need the Orders and OrderItems history.